core/network/src/main/java/com/puzzle/network/source/configure/ConfigureDataSource.kt

```
@@ -16,6 +20,20 @@ class ConfigDataSource @Inject constructor(
 ) {
+
+    init {
+        val configSettings = remoteConfigSettings {
+            minimumFetchIntervalInSeconds = 0
+        }
+        remoteConfig.setConfigSettingsAsync(configSettings)
+        remoteConfig.addOnConfigUpdateListener(object : ConfigUpdateListener {
+            override fun onUpdate(configUpdate: ConfigUpdate) {
+                remoteConfig.activate()
+            }
+            override fun onError(error: FirebaseRemoteConfigException) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)